### ISSUE TITLE
Codes have unhomogenous formats or are missing

### ISSUE THREAD

**Issue by albertocottica** [edgeryders/annotator_store-gem#197]
Some codes have a field for `name`, as separate from the array `names`, ( like https://edgeryders.eu/annotator/codes/7762.json), whereas other have only the latter (like https://edgeryders.eu/annotator/codes/4246.json).

Additionally, some codes are indicated as ancestry of other codes, but are missing. For example: 

https://edgeryders.eu/annotator/codes/4394.json is missing, but is indicated as the parent node of https://edgeryders.eu/annotator/codes/4977.json

Is it possible that, when deleting or merging a node, the ancestry of its children is not updated? @tanius @damingo

**Comment by damingo** [comment]
On my side both codes have a ``names`` array and neither has a ``name`` field. There once was a ``name`` field which was then renamed to ``name_legacy`` when translated names were introduced and is no longer in use and excluded from the JSON api. Maybe you accessed a cached version?

For the missing parent code I checked all codes in the DB and it's luckily just the one you mentioned. Most likely due to the data migrations we performed a couple of weeks ago. I fixed it by putting the 4977 code at the top level.
